src/main/java/com/arpnetworking/clusteraggregator/configuration/EmitterConfiguration.java

```diff
@@ -91,8 +98,24 @@ public Builder setSinks(final List<Sink> value) {
             return this;
         }
 
+        /**
+         * The number of emitters to create in a pool. Sinks are not necessarily also duplicated.
+         * Required. Cannot be null or empty.
```

Comment:
This should not be required, and as the code is written it is not; it defaults to 1 which is backwards compatible.